tools/diffacto/diffacto.xml

```diff
@@ -0,0 +1,230 @@
+<tool id="diffacto" name="Diffacto" version="@TOOL_VERSION@+galaxy@VERSION_SUFFIX@" python_template_version="3.5">
+    <description>Comparative Protein Abundance from Covariation of Peptide Abundances</description>
+    <macros>
+        <token name="@TOOL_VERSION@">1.0.6</token>
+        <token name="@VERSION_SUFFIX@">0</token>
+    </macros>
+    <requirements>
+        <requirement type="package" version="@TOOL_VERSION@">diffacto</requirement>
+    </requirements>
+    <command detect_errors="exit_code"><![CDATA[
+        diffacto -i '$input'
+        #if $db
+            -db '$db'
+        #end if
+        #if $samples
+            -samples '$samples'
+        #end if
+        -reference '$reference'
+        #if $normalize
+            -normalize $normalize
+        #end if
+        #if $farms_mu
+            -farms_mu $farms_mu
+        #end if
+        #if $farms_alpha
+            -farms_alpha $farms_alpha
+        #end if
+        -min_samples $min_samples
+        -impute_threshold $impute_threshold
+        -cutoff_weight $cutoff_weight
+        $use_unique
+        #if $scale == 'log2'
+            -log2 True
+        #else
+            -log2 False
+        #end if
+        $fast
+        -out '$output'
+        #if $mcfdr
+            -mc_out '$mc_out'
+        #end if
+        #if $loadings
+            -loadings_out '$loadings_out'
+        #end if
+    ]]></command>
+    <inputs>
+        <param name="input" argument="-i" type="data" format="csv" label="Peptides abundances">
```

Comment:
where is this file coming from? Can we make it a TSV?

Reply:
@bgruening   pjagtap@umn.edu asked for this tool.   I don't have a sample workflow from him at this time.

diffacto is using pandas.read_csv() for this input.   I tried using  a tabular input, but got an error.  
However, galaxy was able to autoconvert a tabular to csv for the input, so I think that will work for most cases.  
I will add a comment in the help about the auto conversion.

Reply:
could we just do something like `tr '\t' ',' < file.tsv > file.csv` as a first step and avoid this extra job?

Reply:
@bgruening  I incorporated your suggestion